src/VehicleSetup/JoystickConfig.qml

```diff
@@ -382,6 +407,36 @@ SetupPage {
                                     onClicked: _activeJoystick.throttleMode = 0
                                 }
 
+                                Row {
+                                    x:          20
+                                    width:      parent.width
+                                    spacing:    ScreenTools.defaultFontPixelWidth
+                                    visible:    _activeJoystick.throttleMode == 0
+
+                                    QGCCheckBox {
+                                        id:         accumulator
+                                        checked:    _activeJoystick.accumulator
+                                        text:       qsTr("Spring on throttle support")
+
+                                        onClicked:  _activeJoystick.accumulator = checked
+                                    }
+                                }
+
+                                Row {
+                                    x:          20
+                                    width:      parent.width
+                                    spacing:    ScreenTools.defaultFontPixelWidth
+                                    visible:    _activeJoystick.throttleMode == 0
+
+                                    QGCCheckBox {
+                                        id:         deadband
+                                        checked:    _activeJoystick.deadband
+                                        text:       qsTr("Deadband (requires re-calibration)")
```

Comment:
I'd make this stupid-proof and invalid the calibration if deadband is turned on. Either that or always calc deadband and save so it's always there, but just don't use if not turned on.